### ISSUE TITLE
EvilEyeVR

### ISSUE THREAD

**Issue by theRealKLH** [xbapps/xbvr#322]
Now that there is a scraper for VRPORN.com (PR #308). I propose moving EvilEyeVR scraping from SLR (10 scenes) to account for the additional 18 scenes that are missing.

**Comment by SpedNSFW** [comment]
Once that PR is merged, I'll create another PR for it. It shouldn't be moved in the same PR.

**Comment by theRealKLH** [comment]
this was done PR #329

**Comment by SpedNSFW** [comment]
The PR hasn't been merged yet. Re-open this. It will auto-close when it gets merged.

**Comment by theRealKLH** [comment]
yeah. just saw that. was wondering "shouldn't this close automatically?".. GRIN
